### DISCUSSION TITLE
[Q&A - CHAPTER 1] - 자바 8, 9, 10, 11 : 무슨 일이 일어나고 있는가?

### DISCUSSION THREAD

**Discussion by Irisation23** [Study-2-Modern-Java-In-Action/Study-2-Modern-Java-in-Action#19] [CHAPTER 1 - 자바 8, 9, 10, 11 : 무슨 일이 일어나고 있는가? - Q&A]
- 댓글 부탁드려요~

**Comment by BvrPark** [comment]
우선 스터디를 추후에 Kotlin을 배워보고 싶고 자바의 람다식에 대한 필요성을 느껴 스터디를 시작했습니다.

함수형 프로그래밍에 대해 걱정을 하면서 스터디를 시작했지만 다행히 1장에는 앞으로 배울 내용들의 간단한 예제들과 정의들이 나와있어서 앞으로 어떤 내용을 배울 지에 대해서 대략적인 개념을 잡는 데 좋았습니다.

특히 동작 파라미터화라는 개념이 왜 나왔고 그것이 어떻게 람다식으로까지 바뀌는지에 대한 과정이 재밌었습니다. 
2장에서 본격적으로 람다식에 대한 개념들을 더 자세히 공부하고 싶어졌습니다.
